packages/dds/cell/src/cell.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-import { assert , fromBase64ToUtf8 } from "@fluidframework/common-utils";
+import { assert, fromBase64ToUtf8 } from "@fluidframework/common-utils";
```

Comment:
I think all references to fromBase64ToUtf8 should be gone (other than in driver code), as well as all references to read() should be gone as well from runtime code. That would probably be the only way to prove that we did not make a mistake (which is easy to do given code that reads from storage and code that decodes are quite often not next to each other).